gips/data/landsat/landsat.py

```diff
@@ -229,7 +225,7 @@ class landsatAsset(gips.data.core.CloudCoverAsset,
         # DN & SR assets are no longer fetchable
         'DN': {
             'sensors': ['LT5', 'LE7', 'LC8'],
-            'enddate': date(2017, 4, 30),
+            'enddate': date.today(),
```

Comment:
This is wrong.  `DN`-flavored assets (Call it `Collection Zero`) are no longer available for dates after `2017-4-30`. @bhbraswell is there some use-case that needs `.today()` here?

Reply:
Okay, unless I hear otherwise I'll revert to the 2017 date.